### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by brandonyee-cs**
Fixed Grammar

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,13 +61,13 @@ Please feel free to flag any specific need you have need by creating an issue.
 
 Below are some features we have implemented to date:
 
-+ [Add, filter, sort, edit and track customers](#add-filter-sort-edit-and-track-customers)
++ [Add, filter, sort, edit, and track customers](#add-filter-sort-edit-and-track-customers)
```
Comment (charlesBochet) [inline README.md:64]: Hi @brandonyee-cs, I'm not a native english speaker but I believe the grammar was correct here: https://www.grammarly.com/blog/comma-before-and/

File: README.md [hunk 2]
```diff
@@ -26,7 +26,7 @@
 
 We’ve spent thousands of hours grappling with traditional CRMs like Pipedrive and Salesforce to align them with our business needs, only to end up frustrated — customizations are complex and the closed ecosystems of these platforms can feel restrictive.
 
-We felt the need for a CRM platform that empowers rather than constrains. We believe the next great CRM will come from the open source community. And we’ve packed Twenty with powerful features to give you full control and help you run your business efficiently. 
+We felt the need for a CRM platform that empowers rather than constrains. We believe the next great CRM will come from the open-source community. We’ve packed Twenty with powerful features to give you full control and help you run your business efficiently.
```
Comment (charlesBochet) [inline README.md:29]: why not for this one :)

File: README.md (hunk 1 above)
Comment (brandonyee-cs) [reply README.md:64]: Hey, regarding this grammar change, it is actually a widely debated comma when it comes to English. It is called the Oxford Comma, and it is generally considered to be grammatically correct. 

See here: https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/#:~:text=The%20Oxford%20(or%20serial)%20comma,use%20while%20others%20don't.

File: README.md (hunk 2 above)
Comment (brandonyee-cs) [reply README.md:29]: what do you mean by this?

**Review by charlesBochet — APPROVED** [review]
Ok, thanks for the article!

**Pull request merged**